src/supportedLanguages.js

```diff
@@ -121,7 +121,7 @@ module.exports = {
           install: 'npm install axios --save',
         },
         fetch: {
```

Comment:
Yeah this key is tied to the `fetch` target in `httpsnippet` so changing it now would break that.